easybuild/easyconfigs/s/Stata/Stata-18.eb

```diff
@@ -0,0 +1,40 @@
+name = 'Stata'
+version = '18'
+
+homepage = 'https://www.stata.com/'
+description = """Stata is a complete, integrated statistical software package that provides everything you need
+ for data analysis, data management, and graphics."""
+
+toolchain = SYSTEM
+
+download_instructions = f"""{name} requires manual download from https://download.stata.com/download/
+Required download: {' '.join(sources)}
+
+See also https://www.stata.com/support/faqs/unix/install-download-on-linux/"""
+
+build_info_msg = """Once the installation with EB completes, you **must** generate the licence file
+by executing the script "stinit" from the root of the installation directory."""
+
+sources = ['Stata%(version)sLinux64.tar.gz']
+checksums = ['afc02cda8ed5034ab3343b82cbea7bf2139bff7bb2fc1d59d676a104f037d669']
+
+dependencies = [
+    ('ncurses', '5.9'),
+    ('zlib', '1.2.13'),
+]
+
+# Optional: requirements for xstata
+osdependencies = [('xorg', 'xorg-x11', 'xorg-x11-apps', 'xorg-x11-server-Xorg')]
+dependencies.append(
+    ('libpng', '1.6.40'),
+)
```

Comment:
Looks like this should be removed, seems to work fine (on RHEL9 at least) without it:
```suggestion
```